src/games/strategy/triplea/ai/proAI/ProData.java

```diff
@@ -0,0 +1,43 @@
+package games.strategy.triplea.ai.proAI;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.triplea.Properties;
+
+/**
+ * Pro AI data.
+ */
+public class ProData {
+
+  public static double winPercentage = 95;
+  public static double minWinPercentage = 75;
+  public static boolean areNeutralsPassableByAir = false;
+
+  private static GameData data;
+  private static ProAI proAI;
+
+  public static GameData getData() {
+    return data;
+  }
+
+  public static void setData(final GameData data) {
+    ProData.data = data;
+
+    // Set optimal and win percentages lower if not LL
+    if (!games.strategy.triplea.Properties.getLow_Luck(data)) {
+      winPercentage = 90;
+      minWinPercentage = 65;
+    }
+
+    // Set map properties
+    areNeutralsPassableByAir = (Properties.getNeutralFlyoverAllowed(data) && !Properties.getNeutralsImpassable(data));
```

Comment:
Can't make an omelette without breaking some eggs. An alternative, perhaps create an "AIProperties" class to act as an intermediary between the AI code and the core game engine. Then at least you can "fake" the view of the world you wish the AI code had on the engine code.